week-8/Homework/mandatory/3-dog-photo-gallery/script.js

```diff
@@ -0,0 +1,41 @@
+function getImage(){
+fetch ('https://dog.ceo/api/breeds/image/random')
```

Comment:
Cool, this works well!

How would you implement error handling? Remember you can do any DOM manipulation you like, so you can show any messaging to the user...